python/ray/data/block.py

```diff
@@ -166,6 +167,7 @@ class _BlockExecStatsBuilder:
     def __init__(self):
         self.start_time = time.perf_counter()
         self.start_cpu = time.process_time()
+        self.prev_time = None
```

Comment:
can we update this to be more descriptive about what time it exactly represents? e.g. `prev_map_task_finish_time`

Reply:
Yes good point